pkg/edgeadm/common/common.go

```diff
@@ -20,8 +20,10 @@ import (
 	"context"
 	"encoding/base64"
 	"fmt"
+	"github.com/superedge/superedge/pkg/edgeadm/constant/manifests/edgex"
 	"k8s.io/klog/v2"
```

Comment:
Can you put these two lines in the k8s.io and github.com/superedge/superedge import group?